web_pwa_oca/README.rst

```diff
@@ -0,0 +1,134 @@
+===========================
+Progressive web application
+===========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fweb-lightgray.png?logo=github
+    :target: https://github.com/OCA/web/tree/12.0/web_pwa_oca
+    :alt: OCA/web
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/web-12-0/web-12-0-web_pwa_oca
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/162/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+Make Odoo an installable Progressive Web Application.
+
+Progressive Web Apps provide an installable, app-like experience on desktop and mobile that are built and delivered directly via the web.
+They're web apps that are fast and reliable. And most importantly, they're web apps that work in any browser.
+If you're building a web app today, you're already on the path towards building a Progressive Web App.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+After having installed this module, browsing your odoo on mobile you will be able to install it as a PWA.
+
+It is strongly recommended to use this module with a responsive layout, like the one provided by web_responsive.
+
+This module is intended to be used by Odoo back-end users (employees).
+
+When a Progressive Web App is installed, it looks and behaves like all of the other installed apps.
+It launches from the same place that other apps launch. It runs in an app without an address bar or other browser UI.
+And like all other installed apps, it's a top level app in the task switcher.
+
+In Chrome, a Progressive Web App can either be installed through the three-dot context menu.
+
+This module also provides a "Install PWA" link in Odoo user menu.
+
+In case you previously installed `web_pwa`, run the following steps with `odoo shell`, after having installed `openupgradelib`:
+
+
+>>> from openupgradelib import openupgrade
+>>> openupgrade.update_module_names(env.cr, [('web_pwa', 'web_pwa_oca')], merge_modules=False)
```

Comment:
Isto não é mais necessário

Reply:
It is removed from `readme/INSTALL.rst ` and `README.rst` will be re-generated by ocabot